### PR TITLE
Point tests to download/sync from beta-galaxy

### PR DESCRIPTION
**Pull request by rochacbruno**
Temporary change to increase the load on beta-galaxy

No-Issue

### CODE REVIEW

File: galaxy_ng/tests/unit/api/test_api_ui_sync_config.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def test_negative_update_community_repo_data_without_requirements_file(self):
                 "name": "community",
                 "policy": "immediate",
                 "requirements_file": None,
-                "url": "https://galaxy.ansible.com/v3/collections",
+                "url": "https://beta-galaxy.ansible.com/api/v3/collections",
```
[the remark this answers is not in the document]
Comment (awcrosby) [reply galaxy_ng/tests/unit/api/test_api_ui_sync_config.py:104]: IIUC this test only attempts to update the `community` remote, it does not actually attempt to use the remote's `url` to sync, that is likely why it passed with "https://galaxy.ansible.com/v3/collections". I wonder why it failed in this PR earlier even with a non-existent `url`

[the remark this answers is not in the document]
Comment (rochacbruno) [reply galaxy_ng/tests/unit/api/test_api_ui_sync_config.py:104]: It was erroring after the other test that positively changes the URL of the remote, as it was actually changing, subsequent tests were failing to sync.

**Review by awcrosby — APPROVED** [review]

**Pull request merged**